features/data-urls.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-10-21
-name: The "data" URL scheme
-description: TODO
+name: Data URLs
+description: Data URLs are URLs prefixed with the `data:` scheme and embed small files inline in documents.
```

I'm having trouble with the word "files" here. They're not files per say since they're never saved to disk. It's really just inline data + a mime-type.

How about:

```suggestion
description: Data URLs are URLs that start with the `data:` scheme and which directly contain data rather than point to remote resources.
```